notifications.md

```diff
@@ -557,7 +581,7 @@ If needed, you may return a full [mailable object](/docs/{{version}}/mail) from
      * Get the mail representation of the notification.
      *
      * @param  mixed  $notifiable
-     * @return Mailable
+     * @return \Illuminate\Notifications\Messages\MailMessage
```

Comment:
This should still be a Mailable.
Because in this Section it is about returning a Mailable instead of a MailMessage.
```suggestion
     * @return Mailable
```